x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec

```diff
@@ -329,3 +350,154 @@ emp_no:integer | languages:integer | max_salary:integer
          10002 |                 5 | 66817
          10003 |                 4 | 74572
 ;
+
+beforeEnrich
+required_capability: inlinestats
+required_capability: enrich_load
+
+FROM airports
+| KEEP abbrev, type, city
+| INLINESTATS COUNT(*) BY type
+| ENRICH city_names ON city WITH region
+| WHERE MV_COUNT(region) == 1
+| SORT abbrev ASC
+| LIMIT 3
+;
+
+abbrev:keyword | type:keyword |    city:keyword    | "COUNT(*)":long | region:text
+           ABJ |          mid |            Abidjan |             499 | Abidjan
+           ABV |        major |              Abuja |             385 | Municipal Area Council
+           ACA |        major | Acapulco de Juárez |             385 | Acapulco de Juárez
+;
+
+beforeAndAfterEnrich
+required_capability: inlinestats
+required_capability: enrich_load
+
+FROM airports
+| KEEP abbrev, type, city
+| INLINESTATS COUNT(*) BY type
+| ENRICH city_names ON city WITH region
+| WHERE MV_COUNT(region) == 1
+| INLINESTATS count_region=COUNT(*) BY region 
+| SORT abbrev ASC
+| LIMIT 3
+;
+
+abbrev:keyword | type:keyword |    city:keyword    | "COUNT(*)":long | region:text            | count_region:long
+           ABJ |          mid |            Abidjan |             499 | Abidjan                | 1
+           ABV |        major |              Abuja |             385 | Municipal Area Council | 1
+           ACA |        major | Acapulco de Juárez |             385 | Acapulco de Juárez     | 1
+;
+
+
+shadowing
+required_capability: inlinestats
+
+ROW left = "left", client_ip = "172.21.0.5", env = "env", right = "right"
+| INLINESTATS env=VALUES(right) BY client_ip
+;
+
+left:keyword | client_ip:keyword | right:keyword | env:keyword
+left         | 172.21.0.5        | right         | right
+;
+
+shadowingLimit0
+required_capability: inlinestats
+
+ROW left = "left", client_ip = "172.21.0.5", env = "env", right = "right"
+| INLINESTATS env=VALUES(right) BY client_ip
+| LIMIT 0
+;
```

Comment:
I don't think we need the limit0 tests added here; the limit0 tests are only present in `enrich.csv-spec` so we can run at least the logical optimizer against tests that otherwise would need `enrich_load`.